tools/src/test/java/org/apache/kafka/tools/consumer/group/DescribeConsumerGroupTest.java

```diff
@@ -987,6 +987,7 @@ private Map<String, Object> composeConfigs(String groupId, String groupProtocol,
         configs.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class.getName());
         configs.put(ConsumerConfig.GROUP_PROTOCOL_CONFIG, groupProtocol);
         configs.put(ConsumerConfig.PARTITION_ASSIGNMENT_STRATEGY_CONFIG, RangeAssignor.class.getName());
+        configs.put(ConsumerConfig.MAX_POLL_INTERVAL_MS_CONFIG, "1000");
```

Comment:
Pardon me, tot sure why we need to add this config

Reply:
Also, it seems to me `group.consumer.heartbeat.interval.ms` is the true critical config, as they needs some HB cycles to reach the stable group.

Reply:
> Pardon me, not sure why we need to add this config
The document write that 
> A new config, group.initial.rebalance.delay.ms, was introduced. This config specifies the time, in milliseconds, that the GroupCoordinator will delay the initial consumer rebalance. The rebalance will be further delayed by the value of group.initial.rebalance.delay.ms as new members join the group, up to a maximum of max.poll.interval.ms. The default value for this is 3 seconds. During development and testing it might be desirable to set this to 0 in order to not delay test execution time.
This config will speed up in the test

Reply:
The config I didn't catch is `group.initial.rebalance.delay.ms` rather than `max.poll.interval.ms`. `max.poll.interval.ms` the the upper limit of "timeout" and in testing we don't expect to see timeout, so I can't understand the purpose of reducing the value.

Reply:
You are right, the test should not expect timeout, I misunderstanding the `max.poll.interval.ms` config